grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -510,6 +513,177 @@ class ApiController extends BaseController {
         }
     }
 
+    @GrantAccess
```

Comment:
don't think grant access is needed. all access to private collection image must have valid access token.